packages/command/src/docOpts.ts

```diff
@@ -0,0 +1,246 @@
+import { Dictionary, ensure, ensureArray, getString, isArray, isBoolean, isPlainObject } from '@salesforce/ts-types';
+import { definiteEntriesOf } from '@salesforce/ts-types';
+import { get } from '@salesforce/ts-types';
+import { SfdxCommand } from './sfdxCommand';
+import { flags, optionalBuiltinFlags, requiredBuiltinFlags } from './sfdxFlags';
+
+type FlagType = flags.Any<unknown> & { name: string };
+type FlagsType = FlagType[];
+
+/**
+ * DocOpts generator for SfdxCommands.  See http://docopt.org/.
+ *
+ * flag.exclusive: groups elements when one of the mutually exclusive cases is a required flag: (--apple | --orange)
+ * flag.exclusive: groups elements when none of the mutually exclusive cases is required (optional flags): [--apple | --orange]
+ * flag.dependsOn: specifies that if one element is present, then another one is required: (--apple --orange)
+ * cmd.variableArgs: produces 'name=value'
+ *
+ * @example
```

Comment:
I think @example renders ugly in typedoc.  Using triple-backtick blocks seems to work a lot better and is what we've used elsewhere for typedoc.